components/eamxx/src/mct_coupling/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ set (SCREAM_LIBS
      scream_share
      scream_control
      ${dynLibName}
+     mam
```

Comment:
Shouldn't this be guarded by an `if (SCREAM_ENABLE_MAM)` clause?

Reply:
Ah, good eye! I'll fix that.